### ISSUE TITLE
warp.so: undefined symbol: _ZN2wp24hash_grid_rebuild_deviceERKNS_8HashGridEPKNS_4vec3Ei

### ISSUE THREAD

**Issue by lebrice** [NVIDIA/warp#19]
Hey there, just installed the repo, and did the following:
```console
$ conda create -n warp python=3.9
...
$ conda activate warp
$ pip install -e .
...
$ python examples/example_dem.py 
Traceback (most recent call last):
  File "/home/fabrice/Source/warp/examples/example_dem.py", line 24, in <module>
    wp.init()
  File "/home/fabrice/Source/warp/warp/context.py", line 1097, in init
    runtime = Runtime()
  File "/home/fabrice/Source/warp/warp/context.py", line 470, in __init__
    self.core = warp.build.load_dll(warp_lib)
  File "/home/fabrice/Source/warp/warp/build.py", line 285, in load_dll
    dll = ctypes.CDLL(dll_path)
  File "/home/fabrice/miniconda3/envs/warp/lib/python3.9/ctypes/__init__.py", line 382, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /home/fabrice/Source/warp/warp/bin/warp.so: undefined symbol: _ZN2wp24hash_grid_rebuild_deviceERKNS_8HashGridEPKNS_4vec3Ei
```

So then I tried:
```console
$  python build_lib.py 
Namespace(msvc_path=None, sdk_path=None, cuda_path=None, mode='release', verbose=True)
Warning: building without CUDA support
Building /home/fabrice/Source/warp/warp/bin/warp.so
g++ -O3 -DNDEBUG -DWP_CPU  -fPIC --std=c++11 -c "/home/fabrice/Source/warp/warp/native/warp.cpp" -o "/home/fabrice/Source/warp/warp/native/warp.cpp.o"
build took 1178.73 ms
g++ -shared -Wl,-rpath,'$ORIGIN' -o '/home/fabrice/Source/warp/warp/bin/warp.so' "/home/fabrice/Source/warp/warp/native/warp.cpp.o"
link took 72.52 ms
$ python examples/example_dem.py 
Traceback (most recent call last):
  File "/home/fabrice/Source/warp/examples/example_dem.py", line 24, in <module>
    wp.init()
  File "/home/fabrice/Source/warp/warp/context.py", line 1097, in init
    runtime = Runtime()
  File "/home/fabrice/Source/warp/warp/context.py", line 470, in __init__
    self.core = warp.build.load_dll(warp_lib)
  File "/home/fabrice/Source/warp/warp/build.py", line 285, in load_dll
    dll = ctypes.CDLL(dll_path)
  File "/home/fabrice/miniconda3/envs/warp/lib/python3.9/ctypes/__init__.py", line 382, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /home/fabrice/Source/warp/warp/bin/warp.so: undefined symbol: _ZN2wp24hash_grid_rebuild_deviceERKNS_8HashGridEPKNS_4vec3Ei
```
(No change).

OS:  Ubuntu 20.04 LTS
```console
$ gcc --version
gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
```

I'm probably doing something wrong, and I didn't read all the README/documentation thoroughly. But I thought I'd post this, just in case this information might be useful.

Cheers!

**Comment by mmacklin** [comment]
We had a similar issue: https://github.com/NVIDIA/warp/issues/10

This occurs when the lib. is built without CUDA support - we'll update the source to include a stub implementation for this function when CUDA support is disabled.

**Comment by mmacklin** [comment]
This was fixed in commit: e837a66287367c6ecf9cea18c29ee9951a34580d

Thanks,
Miles
